app/scripts/controllers/metametrics-controller.ts

```diff
@@ -175,6 +176,7 @@ export type MetaMaskState = {
     };
   };
   ///: END:ONLY_INCLUDE_IF
+  keyrings: { type: string; accounts: string[] }[];
```

Comment:
Can `keyring.type` by typed (`KeyringType`) or is there a reason to use a `string` here?

Reply:
👍 (reactions are broken on this PR for some reason, lol)